test/factories/answers.rb

```diff
@@ -2,6 +2,6 @@
   factory :answer do
     body { FFaker::HipsterIpsum.sentence }
     question
-    provider { association :user, name: FFaker::Name.name }
+    provider { create(:user) }
```

Comment:
To me,  having the `name:` gave the illusion it was there for a specific reason.